horizomer/utils/tree.py

```diff
@@ -15,6 +15,75 @@
 from skbio import TreeNode
 
 
+def order_nodes(tree, increase=True):
+    """Restore ordering of nodes in one tree based on another tree.
+
+    Parameters
+    ----------
+    tree : skbio.TreeNode
+        tree to order
+    increase : bool, optional
+        order nodes in increasing (True) or decreasing (False) order
+
+    Returns
+    -------
+    skbio.TreeNode
+        resulting ordered tree
```

Comment:
a doctest on how to use this method would be great.

Reply:
@nujinuji doctest is new to me too. If you have time and interest you can explore how this works.